schematics/src/ng-add/index.ts

```diff
@@ -91,17 +91,14 @@ function addScripts({ project }): Rule {
     const workspace = getWorkspace(host);
     const { sourceRoot } = getProjectFromWorkspace(workspace, project || Object.keys(workspace.projects)[0] || 'src');
 
-    asJSON.scripts['generate-icons'] = 'svg-to-ts';
-    asJSON['devDependencies']['svg-to-ts'] = '^5.0.0';
+    asJSON.scripts['generate-icons'] = 'svg-generator';
+    asJSON['devDependencies']['@ngneat/svg-generator'] = '^1.0.0';
 
-    asJSON['svg-to-ts'] = {
-      generateType: 'false',
-      delimiter: 'KEBAB',
-      conversionType: 'files',
-      iconsFolderName: 'svg',
+    asJSON['svgGenerator'] = {
```

Comment:
Update the README, and add Webpack plugin section:

```ts
const {
  SvgGeneratorWebpackPlugin,
} = require('@ngneat/svg-generator/webpack-plugin');

      new SvgGeneratorWebpackPlugin({
        watch: !isProd,
        srcPath: './apps/platform/src/assets/svg',
        outputPath: './packages/svg',
        svgoConfig: {
          plugins: [
            {
              removeDimensions: true,
              cleanupAttrs: true,
            },
          ],
        },
      }),
```

Reply:
Maybe you better write the docs for it?
I'm not sure what to write about it. Is this usage goes instead of CLI? If using this do you need to add config to `package.json`? and is it recommended flow then?